extras/pom.xml

```diff
@@ -44,8 +44,6 @@ under the License.
         <module>rya.export</module>
         <module>rya.merger</module>
         <module>rya.giraph</module>
-       <module>rya.benchmark</module>
-       <module>rya.geoindexing</module>
```

Comment:
Note that the geo module is already correctly removed in another PR.  I needed to test these together.